packages/application-config/src/process-config.ts

```diff
@@ -68,11 +68,16 @@ const processConfig = ({
       ),
     `Invalid MC API URL: "${appConfig.mcApiUrl}"`
   );
+  // TODO: make the `id` required once we release the new model.
```

Comment:
the `id` wil come from DB and is not user provided, right?

Reply:
You have to provide it in the app config.

Reply:
ok. clear